backends/magma/ceed-magma-operator.c

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory. LLNL-CODE-734707.
+// All Rights reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+#include <ceed-backend.h>
+#include "../cuda/ceed-cuda.h"
+#include "ceed-magma.h"
+#include <string.h>
+
+// Create a cuda-ref operator and modify its e_and_q_layout_diff value
+int CeedOperatorCreate_Magma(CeedOperator op) {
+
```

Comment:
extra line

Reply:
Fixed in 3026fe3dfa6c1d826d17c138bee5c49103da2abb